.drone.star

```diff
@@ -1023,7 +1023,7 @@ def acceptance(ctx):
         "oc10IntegrationAppIncluded": False,
         "skip": False,
         "debugSuites": [],
-        "earlyFail": True,
+        "earlyFail": False,
```

Comment:
@kiranparajuli589 we don't need to do this any more - you can put `full-ci` in the PR title and that will force `earlyFail` `False`